js/models/addressBook_model.js

```diff
@@ -13,9 +13,56 @@ app.factory('AddressBook', function()
 					}
 				}
 				return undefined;
+			},
+
+			sharedWith: {
+				users: [],
+				groups: []
 			}
 
 		});
 		angular.extend(this, data);
+		angular.extend(this, {
+			owner: data.url.split('/').slice(-3, -2)[0]
```

Comment:
@DeepDiver1975 Is this ok (with regards to how I have found the owner?). Calendar uses the share prop that you just implemented, is that safer?